packages/xchain-util/src/asset.ts

```diff
@@ -33,9 +35,9 @@ export const assetAmount = (value: string | number | BigNumber | undefined, deci
 /**
  * Factory to create base amounts (e.g. tor)
  *
- * @param value - Base amount - If the value is undefined, BaseAmount with value `0` will be returned
- * @param decimal - Decimal places - default 8
- *
+ * @param {string|number|BigNumber|undefined} value - The base amount, If the value is undefined, BaseAmount with value `0` will be returned.
+ * @param {number} decimal (optional) The decimal places. Default value is 8.
```

Comment:
An `AssetAmount` won't never have any decimal, but its associated `AssetAmount`. That's why I would suggest following change: `The decimal places.` -> `The decimal places of its associated `AssetAmount`.`